src/current/_includes/releases/v23.2/v23.2.4.md

```diff
@@ -0,0 +1,54 @@
+## v23.2.4
+
+Release Date: April 8, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-4-sql-language-changes">SQL language changes</h3>
+
+- Mutation statements such as [`UPDATE`](../v23.2/update.html) and [`DELETE`](../v23.2/delete.html) as well as locking statements such as [`SELECT FOR UPDATE`](../v23.2/select-for-update.html) are not allowed in [read-only transactions](../v23.2/set-transaction.html#parameters) or [`AS OF SYSTEM TIME` transactions](../v23.2/set-transaction.html#parameters). Fixed a bug where mutation statements and locking statements in implicit single-statement transactions using AS OF SYSTEM TIME were allowed. [#120158][#120158]
```

Comment:
```suggestion
- Mutation statements such as [`UPDATE`](../v23.2/update.html) and [`DELETE`](../v23.2/delete.html) as well as locking statements such as [`SELECT FOR UPDATE`](../v23.2/select-for-update.html) are not allowed in [read-only transactions](../v23.2/set-transaction.html#parameters) or [`AS OF SYSTEM TIME` transactions](../v23.2/set-transaction.html#parameters). This is the result of fixing a bug where mutation statements and locking statements in implicit single-statement transactions using `AS OF SYSTEM TIME` were allowed. [#120158][#120158]
```

Reply:
This is kind of a wordy suggestion, but I feel like the relationship between the two sentences wasn't clear. Feel free to adjust further.

Reply:
updated